Provision/seeds/seedData.sql

```diff
@@ -2,7 +2,8 @@ INSERT INTO lol_friend_compare.users (username, email, password, last_login, cre
 ('Cisco', 'franciscoadriangarcia1@gmail.com', '$2y$10$s3Joj9oP.60v62.6P.q7vO9WltC1zvK/vUIgL6/pq44p/NrsSKJ8y', '2016-12-21 04:30:12', '2016-12-21 03:34:44', '2016-12-21 04:30:12');
 
 INSERT INTO player_icon(`id`, `image`) VALUES
-(778, 'http://ddragon.leagueoflegends.com/cdn/6.21.1/img/profileicon/778.png');
+(778, 'http://ddragon.leagueoflegends.com/cdn/6.21.1/img/profileicon/778.png'),
+(984, 'http://ddragon.leagueoflegends.com/cdn/6.21.1/img/profileicon/984.png');
```

Comment:
Makes all the logic, going for the storing as INT.
 - removed unnecessary table
 - removed unnecessary foreign key
 - removed unnecessary seed data